providers/class.two-factor-email.php

```diff
@@ -79,6 +79,23 @@ public function user_has_token( $user_id ) {
 	}
 
 	/**
+	 * Get the authentication token for the user.
+	 *
+	 * @param  int $user_id    User ID.
+	 *
+	 * @return string|boolean  User token or `false` if no token found.
+	 */
+	public function get_user_token( $user_id ) {
+		$hashed_token = get_user_meta( $user_id, self::TOKEN_META_KEY, true );
+
+		if ( ! empty( $hashed_token ) && is_string( $hashed_token ) ) {
```

Comment:
With `$single` being specified as `true` is there any way the returned data won't be a string?  Even if it's a number, wouldn't it be cast as a string?

Reply:
@georgestephanis This is mainly to protect from returning anything but a string or `false`.